### DISCUSSION TITLE
specific area on detect an object

### DISCUSSION THREAD

**Discussion by shprk** [ultralytics/yolov5#5955] [Q&A] [answered]
Is it possible to mark a specific area on the web camera and detect objects within that area?

Please let me know,,

**Comment by glenn-jocher** [comment, marked as answer]
@seonghoon93 the proper workflow is to crop any area of the image you want and pass that crop to the model for detection.

**Reply by shprk** [reply]
Thank you so much for your reply!  You are nice guy! 😄

I'm beginner who just started. I want to draw a box in a specific area.
could you tell me about the Python file and function that i have to fix in yolov5?
or How to crop any area of the image and pass that crop to the model for detection?
Please let me know how to make code... 
oh.. For your information, I want to crop the video from the live webcam.

sorry.. I'm not English very well.. 
Thank you!!

**Reply by shokirxon** [reply]
@seonghoon93 were you able to figured out running detection only on specific area of the frame?

**Reply by Andy824** [reply]
@glenn-jocher Can we use two models at the same time when connecting the webcam? I use model1 to detect an object to generate some of the bounding box areas as a detection area for my model2.

**Reply by animeesh** [reply]
is the code available for this?
